xml/product-entities.ent

```diff
@@ -102,3 +102,15 @@
 <!ENTITY license "Creative Commons Attribution-ShareAlike 4.0 International">
 <!ENTITY license-acronym "CC-BY-SA 4.0">
 <!ENTITY license-url "https://creativecommons.org/licenses/by-sa/4.0/legalcode">
+
+<!-- REUSED PHRASES -->
+<!ENTITY salt_log_level_note '
+<note>
+ <title><option>log_level_logfile</option> and <option>log_level</option></title>
+ <para>
+  While <option>log_level</option> controls which log messages will be displayed
+  on the screen, <option>log_level_logfile</option> controls which log messages
+  will be written to <filename>/var/log/salt/master</filename>.
```

Comment:
There are two daemons here: salt-master and salt-minion. The former writes to logfile `/var/log/salt/master` and the latter to `/var/log/salt/minion`. Each daemon has a configuration file, and inside that configuration file each one has a `log_level_logfile` parameter. So if we are going to reuse this phrase for both daemons, we cannot quote a logfile that applies only to one of them, right?

Reply:
@smithfarm  After i removed the snippet about salt-master as `warning` was the default behaviour, does it make sense to mention the `log_level` option in the salt-minion sentence at all?

Reply:
Yes, I think the info box is still needed because readers are likely to be confused about this. This potential for confusion is there for two reasons:

1. because `log_level` appears in the configuration file first, and
2. because in previous versions of Salt, there was no `log_level_logfile` parameter (the `log_level` parameter used to control both logging to screen and logging to file)

For these two reasons, we have to assume that some readers will overlook that they are changing the `log_level` parameter instead of `log_level_logfile`.